frontend/app/views/spree/taxons/_subcategories.html.erb

```diff
@@ -1,16 +1,24 @@
-<% if taxon.children.any? %>
-  <div class="d-flex justify-content-center position-sticky sticky-top-90px taxon-subcategories-wrapper">
-    <div class="py-1 text-uppercase d-none d-xl-flex overflow-x taxon-subcategories">
-      <% taxon.children.to_a.unshift(taxon).each_with_index.map do |subcategory, index| %>
-        <%=
-          link_to_unless subcategory == taxon,
-                         index.zero? ? Spree.t(:all) : subcategory.name,
-                         spree.nested_taxons_path(subcategory, params: permitted_product_params),
-                         class: "pt-4 pb-2 mb-2 #{index.zero? ? '' : 'pl-4 pt-4 pb-2'}" do
-        %>
-          <div class="pt-4 pb-2 mb-2 <%= index.zero? ? '' : 'pl-4' %> taxon-subcategories-selected"><%= index.zero? ? Spree.t(:all) : subcategory.name %></div>
+<% [parent_taxon, taxon].map do |current_taxon| %>
+  <% child_taxons = current_taxon.children.to_a.unshift(current_taxon) %>
```

Comment:
Wouldn't it be cleaner to use the splat operator here? I see that `to_a` prevents `children` modification but it's not clear at first glance.